awswrangler/_utils.py

```diff
@@ -897,3 +898,21 @@ def list_to_arrow_table(
         arrays.append(v)
     # Will raise if metadata is not None
     return pa.Table.from_arrays(arrays, schema=schema, metadata=metadata)
+
+
+def get_emr_integer_version(emr_version: str, default: int = 670) -> int:
```

Comment:
Two suggestions:
1. Since this function is EMR specific, I would just move it to the emr.py module to avoid bloating our utils
2. If the only use case for this private function is to determine whether to use log4j vs log4j2 then perhaps it should just return that instead of the emr version? We can always refactor and expand its use in the future if need be.

Reply:
ok I will do the changes then.